tests/orders/tests-stats.php

```diff
@@ -173,6 +173,48 @@ public function test_get_order_item_refund_count_with_range_this_year_should_be_
 		$this->assertSame( 2, $count );
 	}
 
+	/**
+	 * @covers ::get_order_item_count
+	 */
+	public function test_get_order_item_count_null_price_id_should_be_2() {
```

Comment:
@robincornett Let's also get a unit test around a `0` price ID. We've captured `null` and `2` but we haven't done a check to make sure `0` still works.